tesla_powerwall/api.py

```diff
@@ -19,12 +17,10 @@ def __init__(
         timeout: int = 10,
         http_session: Optional[aiohttp.ClientSession] = None,
         verify_ssl: bool = False,
-        disable_insecure_warning: bool = True,
     ) -> None:
-        if disable_insecure_warning:
-            disable_warnings(InsecureRequestWarning)
-
-        self._endpoint = self._parse_endpoint(endpoint)
+        if not endpoint.startswith("http"):
+            endpoint = f"https://{endpoint}"
```

Comment:
YARL will refuse to change schema for it doesn't already have one

Reply:
The HA integration passes only the address so I didn't want to make this breaking